shared/common-adapters/input.native.js

```diff
@@ -30,6 +30,10 @@ class Input extends Component<void, Props, State> {
     }
   }
 
+  setNativeProps (props: Object) {
+    this._input && this._input.setNativeProps(props)
```

Comment:
:sheep: It might be preferable to not expose `setNativeProps` and hide it as an implementation detail of an `inputClear` method on the input, so we could have something similar for desktop.